api/api.go

```diff
@@ -269,6 +352,9 @@ func (a *API) Handlers() http.Handler {
 
 	// CLI
 	r.Handle("/", appHandler(a.CLIHandler)).Methods("GET").MatcherFunc(cliMatcher)
+	if a.TestIP {
+		r.Handle("/test/{url}", appHandler(a.TestHandler)).Methods("GET").MatcherFunc(cliMatcher)
```

Comment:
Since it should only test the clients IP, this should take a port parameter instead.